apix-impl/src/test/java/eu/xenit/apix/tests/search/FtsNodeVisitorTest.java

```diff
@@ -57,6 +63,42 @@ public void testIntTypeValid() {
                 containsString("{tenant.model}intProperty"));
     }
 
+    @Test
+    public void testIsUnsetTerm() {
+        String ftsQuery = convertToFtsTerm(IS_UNSET);
```

Comment:
Nice short tests :+1: 
You could even write `assertEquals("ISUNSET:\"prefix:test_prop\"", convertToFtsTerm(IS_UNSET));`, but that's a matter of preference and you don't have to change it.